WPFHospitalEditor/ScheduleWindow.xaml.cs

```diff
@@ -24,25 +23,23 @@ public ScheduleWindow(RecommendationDto recommendationDto)
         {
             InitializeComponent();
             this.recommendationDto = recommendationDto;
-            setAppointmentInfoContent();
-            allPatients = patientServerController.GetAllPatients().ToList();
-            foreach (Patient p in allPatients)
-                patients.Items.Add(p.Id.ToString() + " " + p.Person.Name + " " + p.Person.Surname + " - " + p.Person.Jmbg);
+            SetAppointmentInfoContent();
+            SetPatientNameNameComboBox();
```

Comment:
I'm not sure which value textbox has by default, but it can be empty string, so you will pull all patients on start. Maybe it's better to call setPatientNameCombo method only on textbox changing.

Reply:
I am not sure if I understood this comment properly. If I did, user doesn't need to write something in textbox, he can just choose from the combobox, and at the beginning there must be a full list of patients in the combobox. Also, now I see name of method, it was accident, it is fixed now.

Reply:
If the idea is that user can choose from combo on beginning, that's ok then.